tests/language.spec.js

```diff
@@ -0,0 +1,33 @@
+const axios = require('axios');
+const { URL } = require('../constants');
+const getToken = require('./functions/getToken');
+
+let token;
+beforeAll(async () => {
+  token = await getToken();
+});
+
+describe('language resolvers', () => {
+  test('updateLanguage', async () => {
+    const response = await axios.post(
+      URL,
+      {
+        query: `
+            mutation {
+                updateLanguage(languageCode: "es") {
```

Comment:
Please create an array variable containing language code, then from that array randomly select one language code and use that variable as an input here